### DISCUSSION TITLE
2024-05-31(금)

### DISCUSSION THREAD

**Discussion by Seoin02** [Together-3team/petFrontend#72] [데일리스크럼]
아래 내용을 복사해서 댓글에 작성해주시면 됩니다.

```
### 완료한 일

### 해야할 일

### 어려운 일
```

**Comment by Seoin02** [comment]
### 완료한 일
- signup 페이지 UI 구현(일부 이슈빼고 완성)

### 해야할 일
- onboarding 페이지 UI 구현
- OAuth 스웨거 확인

### 어려운 일
- signup form 에러메시지 디자인
- 버튼 공통 컴포넌트 크기 수정, type 추가
- 체크박스 CSS

**Comment by devwqc** [comment]
### 완료한 일
- AWS 메모리, 디스크 용량 문제 해결
- 배너 캐러셀 작업 시작

### 해야할 일
- 배너 캐러셀 컴포넌트 구현
- 루트 레이아웃 추가 작업
- 카테고리 버튼 컴포넌트 구현
- 스크롤 Top 버튼 컴포넌트 구현

### 어려운 일
- 송은, 민영, 지원, 서인 파이팅!

**Comment by q45402sk** [comment]
### 완료한 일

### 해야할 일
zzim 컴포넌트 

### 어려운 일

**Comment by minyoung0503** [comment]
### 완료한 일
- 상품 상세 페이지 -> 리뷰 layout

### 해야할 일
- 리뷰 전체보기 페이지 작업
- 별점 컴포넌트 수정
- 프로필 기본 이미지 수정

### 어려운 일
